src/applications/caregivers/utils/helpers.js

```diff
@@ -1,150 +1,112 @@
-import { isEmpty, mapValues } from 'lodash';
-import caregiverFacilities from 'vets-json-schema/dist/caregiverProgramFacilities.json';
+import { camelCase, isEmpty, omit } from 'lodash';
 import { transformForSubmit } from 'platform/forms-system/src/js/helpers';
-import {
-  primaryCaregiverFields,
-  secondaryOneFields,
-} from '../definitions/constants';
-
-// Merges all the state facilities into one object with values as keys
-// and labels as values
-export const medicalCenterLabels = Object.keys(caregiverFacilities).reduce(
-  (labels, state) => {
-    const stateLabels = caregiverFacilities[state].reduce(
-      (centers, center) =>
-        Object.assign(centers, {
-          [center.code]: center.label,
-        }),
-      {},
-    );
-
-    return Object.assign(labels, stateLabels);
-  },
-  {},
-);
-
-// Turns the facility list for each state into an array of strings
-export const medicalCentersByState = mapValues(caregiverFacilities, val =>
-  val.map(center => center.code),
-);
-
-// transforms forData to match fullSchema structure for backend submission
-export const submitTransform = (formConfig, form) => {
-  const hasPrimary = form.data['view:hasPrimaryCaregiver'] ? 'primary' : null;
 
-  const hasSecondaryOne = form.data['view:hasSecondaryCaregiverOne']
+// transforms form data to match fullSchema structure for backend submission
+export const submitTransform = (formConfig, form) => {
+  const { data: formData } = form;
+  const hasPrimary = formData['view:hasPrimaryCaregiver'] ? 'primary' : null;
```

Comment:
I really don't like these names, but can't thing of anything better. The name makes sense for the use, but not when you look at what the value actually is.